spec/index.html

```diff
@@ -2627,6 +2627,46 @@ <h3>Algorithm</h3>
   </section>
 </section>
 
+<section id="serialization">
+  <h2>Serialization</h2>
+
+  <p>This section describes the process of creating a serialized [[N-Quads]] representation
+    of a <a>normalized dataset</a>.</p>
+
+  <p>The <dfn class="lint-ignore">serialized canonical form</dfn> of a <a>normalized dataset</a>
+    is an <a data-cite="N-QUADS#n-quads-document">N-Quads document</a>
+    created by representing each <a>quad</a> from the <a>normalized dataset</a>
+    in <a>canonical n-quads form</a>,
+    sorting them into <a>code point order</a>,
+    and concatenating them.
+    The resulting document document has a media type of `application/n-quads`,
+    as described in <a data-cite="N-QUADS#sec-mediaReg">C. N-Quads Internet Media Type, File Extension and Macintosh File Type</a>
+    of [[N-QUADS]].</p>
+
+  <p>When serializing quads in <a>canonical n-quads form</a>
+    components which are <a>blank nodes</a> MUST be serialized using the
+    canonical label associated with the <a>blank node</a> in the <a>normalized dataset</a>.</p>
+
+  <aside id="ex-ser-unique-hashes"
+        class="example"
+        title="Canonical N-Quads representation of Unique Hashes">
+
+    <p>This example illustrates the result of serializing
+      the normalized dataset described in <a href="#ex-ca-unique-hashes"></a>.</p>
+
+    <pre id="ex-ser-unique-hashes-result"
+         data-transform="updateExample"
+         title="Canonical N-Quads representation of Unique Hashes">
+    <!--
+      <http://example.com/p> <http://example.com/q> _:c14n0 .
+      <http://example.com/p> <http://example.com/r> _:c14n1 .
+      _:c14n0 <http://example.com/s> <http://example.com/u> .
+      _:c14n1 <http://example.com/t> <http://example.com/u> .
```

Comment:
```suggestion
      _:c14n1 <http://example.com/t> <http://example.com/u> .
      
```

While I want to explicitly represent **the final EOL** of canonical N-Quads, I am wondering this newline can be correctly handled by the document parser.

Reply:
An N-Triples parser will certainly be able to handle extra whitespace, and the example can't really be displayed with the exact whitespace as it's embedded in styling. That last "blank line" is really just the HTML box model with borders. Removing all trailing whitespace (which would not be canonical) still shows empty whitespace.